import-issues.py

```diff
@@ -192,14 +197,18 @@ def __init__(self, card_data, args, labels_mapper):
 
     @property
     def state(self):
+
```

Comment:
extra line between function header and body doesn't align with the rest of the code style